x/mongo/driver/driver.go

```diff
@@ -88,11 +88,24 @@ type Compressor interface {
 	CompressWireMessage(src, dst []byte) ([]byte, error)
 }
 
+// ProcessErrorResult represents the result of a ErrorProcessor.ProcessError() call.
+type ProcessErrorResult int
+
+const (
+	// NoChange indicates that the error did not affect the state of the server.
+	NoChange ProcessErrorResult = iota
+	// ServerMarkedUnknown indicates that the error only resulted in the server being marked as Unknown.
+	ServerMarkedUnknown
+	// ConnectionPoolCleared indicates that the error resulted in the server being marked as Unknown and its connection
+	// pool being cleared. This is a superset of ServerMarkedUnknown.
```

Comment:
I think this should say "subset". A ConnectionPoolCleared event implies a ServerMarkedUnknown event, but not vice versa.

Reply:
Discussed offline. Removed the word "superset" because it makes the docs confusing.